esteid/actions.py

```diff
@@ -1,4 +1,3 @@
-# pragma: no cover
```

Comment:
Can apply to code blocks, but not the whole file. Whole file can be excluded in `setup.cfg`